postman-generator/src/main/scala/models/attributes/PostmanAttributes.scala

```diff
@@ -1,17 +1,63 @@
 package models.attributes
 
-import io.flow.postman.generator.attributes.v0.models.ObjectReference
+import io.flow.postman.generator.attributes.v0.models._
+import play.api.libs.json._
+import io.flow.postman.generator.attributes.v0.models.json._
+import io.apibuilder.spec.v0.models.json._
+
+import scala.language.implicitConversions
+import scala.util.Random
 
 object PostmanAttributes {
 
-  def postmanVariableNameFrom(objReference: ObjectReference): String = {
+  case class ExtendedObjectReference(
+    relatedServiceNamespace: String,
+    resourceType: String,
+    operationMethod: io.apibuilder.spec.v0.models.Method,
+    identifierField: String,
+    path: Option[String] = None
+  )
+
+  implicit class ModelReferenceExtend(m: ModelReference) {
+
+    def toExtended: ExtendedObjectReference = ExtendedObjectReference(
+      relatedServiceNamespace = m.relatedServiceNamespace,
+      resourceType = m.resourceType,
+      operationMethod = m.operationMethod,
+      identifierField = m.identifierField
+    )
+  }
+
+  implicit class PathReferenceExtend(p: PathReference) {
+
+    def toExtended: ExtendedObjectReference =
+      ExtendedObjectReference(
+        relatedServiceNamespace = p.relatedServiceNamespace,
+        resourceType = p.resourceType,
+        operationMethod = p.operationMethod,
+        identifierField = p.identifierField,
+        path = p.path
+      )
+  }
+
+  def postmanVariableNameFrom(objReference: ExtendedObjectReference): String = {
     import objReference._
     s"$resourceType#$identifierField"
   }
 
-  def postmanVariableRefFrom(objectReference: ObjectReference): String = {
+  def postmanVariableRefFrom(objectReference: ExtendedObjectReference): String = {
     val variableName = postmanVariableNameFrom(objectReference)
     s"{{$variableName}}"
   }
 
+  val BasicAuthKey = "postman-basic-auth"
+  val SetupKey = "postman-organization-setup"
+  val ObjectReferenceKey = "object-reference"
+
+  final case class PostmanBasicAuthAttrValue(
```

Comment:
basic auth entity should be already auto-generated from ApiBuilder specification